### ISSUE TITLE
how to use elevation_map_loader in perception?

### ISSUE THREAD

**Issue by LLplane** [autowarefoundation/autoware.universe#539]
Hello!
What role does "elevation_map_loader" play in perception?
And how to use it?

Think U !

**Comment by kenji-miyake** [comment]
@LLplane Ah, I'm sorry. since it was impossible to convert from different repositories, could you ask again?

**Comment by kenji-miyake** [comment]
I've updated the settings of issue templates. https://github.com/autowarefoundation/autoware.universe/pull/540

So now you can select a type.
![image](https://user-images.githubusercontent.com/31987104/158788228-be5bfe88-3d16-4a61-af6e-6d6915e7f8c0.png)

**Comment by LLplane** [comment]
"elevation_map_loader" is new node in aw.universe,and I do not know how to use it. I select Question to ask.

**Comment by kenji-miyake** [comment]
Yes, if you open a new question thread, I will `@mention` the author, and he would explain it.

**Comment by LLplane** [comment]
Sorry, I do not know how to open a new question, can not find the page like the above picture.

**Comment by kenji-miyake** [comment]
Here! Just click the `New issue` button.

![image](https://user-images.githubusercontent.com/31987104/158789903-1c3fae5c-f746-490d-9fa6-11f197dd9942.png)

**Comment by LLplane** [comment]
ooo,think u !
